src/doc/build-script.md

```diff
@@ -294,19 +298,21 @@ the build script now:
 ```rust,no_run
 // build.rs
 
-use std::io::Command;
-use std::os;
+use std::process::Command;
+use std::env;
 
 fn main() {
-    let out_dir = os::getenv("OUT_DIR").unwrap();
+    let out_dir = env::var_os("OUT_DIR").unwrap()
```

Comment:
Instead of using `into_string` this may be better expressed as just `env::var` instad.